src/features/import/hooks/useGuessOrganisation.ts

```diff
@@ -0,0 +1,53 @@
+import Fuse from 'fuse.js';
+
+import { UIDataColumn } from './useUIDataColumns';
+import useSubOrganizations from 'features/organizations/hooks/useSubOrganizations';
+import { CellData, OrgColumn } from '../utils/types';
+
+type OrgMap = { orgId: number; value: CellData };
+
+const useGuessOrganisaion = (
```

Comment:
There is a spelling mistake here (missing "t"), but also when picking between the two empires I tend to go with the more recent one and prefer "z" over "s". 😞